data/json/recipes/recipe_food.json

```diff
@@ -9234,6 +9237,79 @@
   ]
 },{
   "type" : "recipe",
+  "result": "sauerkraut_ferment",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_VEGGI",
+  "skill_used": "cooking",
+  "difficulty": 5,
+  "time": 30000,
+  "reversible": false,
+  "autolearn": true,
+  "qualities" : [
+    {"id":"CUT","level":1,"amount":1},
+    {"id":"COOK","level":3,"amount":1}
+  ], "tools": [
+    [
+      [ "hotplate", 20 ],
+      [ "char_smoker", 1 ],
+      [ "toolset", 20 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "water", 1 ],
+      [ "water_clean", 1 ]
+    ],
+    [
+      [ "salt_water", 2 ],
+      [ "saline", 5 ],
+      [ "salt", 1]
+    ],
+    [
+      [ "jar_glass", 1 ]
+    ],
+    [
+      ["irradiated_lettuce", 2],
```

Comment:
The irradiated stuff wouldn't work, since the entire fermentation process relies on naturally occurring microbes found on the surface of fresh plants.

How about allowing stuff like wild vegetables or plant marrow as options as well?